### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by hamishwillee** [mdn/browser-compat-data#14736]
I've just returned from holiday and am getting a build error when I run `yarn test` on main branch. Note that this is a completely clean clone - all I have done is cloned, run `yarn`, run `yarn test`. All worked before I left.

The error output is as shown:
```
  28 passing (1s)
  1 failing

  1) release-build
       pre-built bundles are identical to the source:
     Command failed: npm run release-build
TypeError: fs.rm is not a function
    at main (D:\github\mozilla\browser-compat-data\scripts\release-build.js:75:6)
    at Object.<anonymous> (D:\github\mozilla\browser-compat-data\scripts\release-build.js:97:1)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @mdn/browser-compat-data@4.1.5 release-build: `node scripts/release-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @mdn/browser-compat-data@4.1.5 release-build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\hamis.000\AppData\Roaming\npm-cache\_logs\2022-01-31T02_42_18_129Z-debug.log

  Error: Command failed: npm run release-build
  TypeError: fs.rm is not a function
      at main (scripts\release-build.js:75:6)
      at Object.<anonymous> (scripts\release-build.js:97:1)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
      at internal/main/run_main_module.js:17:47
  npm ERR! code ELIFECYCLE
  npm ERR! errno 1
  npm ERR! @mdn/browser-compat-data@4.1.5 release-build: `node scripts/release-build`
  npm ERR! Exit status 1
  npm ERR!
  npm ERR! Failed at the @mdn/browser-compat-data@4.1.5 release-build script.
  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

  npm ERR! A complete log of this run can be found in:
  npm ERR!     C:\Users\hamis.000\AppData\Roaming\npm-cache\_logs\2022-01-31T02_42_18_129Z-debug.log

      at checkExecSyncError (child_process.js:635:11)
      at execSync (child_process.js:671:15)
      at Context.<anonymous> (scripts\release-build.test.js:9:5)
      at processImmediate (internal/timers.js:461:21)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @mdn/browser-compat-data@4.1.5 mocha: `mocha "**/*.test.js"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @mdn/browser-compat-data@4.1.5 mocha script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\hamis.000\AppData\Roaming\npm-cache\_logs\2022-01-31T02_42_18_969Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

D:\github\mozilla\browser-compat-data>
```

**Comment by queengooborg** [comment]
I'm surprised that Yarn was working for you before, but we're using NPM for this project.  Please make sure to use the `npm` command instead!

**Comment by hamishwillee** [comment]
I'm an idiot. Thanks.

FYI, it does work except for the last step.

**Comment by queengooborg** [comment]
All good, it gets even the best of us sometimes!

I think one of the big parts is that BCD is the only MDN project (at least, out of the ones I've seen) that still uses NPM.  I opened up an issue to discuss switching to Yarn a while back: https://github.com/mdn/browser-compat-data/issues/12137

**Comment by hamishwillee** [comment]
@queengooborg Actually, this also happens for me with npm, on windows. So can you please reopen.

**Comment by hamishwillee** [comment]
@queengooborg Thanks. Yes, that is the cause of this problem. Pity the tools don't warn of minimal version :-(

I'll close this issue.

FYI I do still get errors on 14.14.0, but they are different. 

```
28 passing (2s)
  1 failing

  1) spec_url data
       spec_urls only use allow listed hosts by w3c/browser-specs (and our exception list):

      AssertionError [ERR_ASSERTION]: Invalid specification host(s) found. Try a more current specification URL and/or
      check if the specification URL is listed in https://github.com/w3c/browser-specs.
      + expected - actual

      -[
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-asynciterable"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-getdirectoryhandle"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-getfilehandle"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-asynciterable"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-removeentry"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-resolve"
      -  "https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle-asynciterable"
      -  "https://fs.spec.whatwg.org/#api-filesystemfilehandle"
      -  "https://fs.spec.whatwg.org/#api-filesystemfilehandle-createwritable"
      -  "https://fs.spec.whatwg.org/#api-filesystemfilehandle-getfile"
      -  "https://fs.spec.whatwg.org/#api-filesystemhandle"
      -  "https://fs.spec.whatwg.org/#api-filesystemhandle-issameentry"
      -  "https://fs.spec.whatwg.org/#ref-for-dom-filesystemhandle-kind①"
      -  "https://fs.spec.whatwg.org/#ref-for-dom-filesystemhandle-name①"
      -  "https://fs.spec.whatwg.org/#api-filesystemwritablefilestream"
      -  "https://fs.spec.whatwg.org/#api-filesystemwritablefilestream-seek"
      -  "https://fs.spec.whatwg.org/#api-filesystemwritablefilestream-truncate"
      -  "https://fs.spec.whatwg.org/#api-filesystemwritablefilestream-write"
      -  "https://fs.spec.whatwg.org/#dom-storagemanager-getdirectory"
      -]
      +[]

      at Context.<anonymous> (test\spec-urls.test.js:64:12)
      at processImmediate (internal/timers.js:461:21)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @mdn/browser-compat-data@4.1.14 mocha: `mocha "**/*.test.js"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @mdn/browser-compat-data@4.1.14 mocha script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\hamis.000\AppData\Roaming\npm-cache\_logs\2022-04-03T23_57_16_552Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by queengooborg** [comment]
Glad to hear that one problem was resolved!  I opened up #11335 to help with that, but there hasn't been much activity on infrastructure PRs.

In regards to the browser specs error, rerunning `npm i --save-dev` should fix that!
